QuickViewFile/MainWindowNoBorder.xaml

```diff
@@ -50,15 +50,24 @@
               Padding="10,10,10,10"
               TextWrapping="NoWrap" />
 
-            <Button
-              Grid.Column="1"
-              Content="Save"
-              Name="SaveButton"
-              Height="24"
-              Margin="0,8,10,8"
-              Padding="15,0"
-              Click="SaveButton_Click"
-              Visibility="{Binding SelectedItem.FileContentModel.ShowTextBox, Converter={StaticResource BoolToVisibilityConverter}}"/>
+            <StackPanel Grid.Column="1" Orientation="Horizontal" HorizontalAlignment="Right">
+                <Button
+                  Content="HQ"
+                  Name="HQButton"
+                  Height="24"
+                  Margin="0,8,10,8"
+                  Padding="10,0"
+                  Click="HQButton_Click"
+                  ToolTip="Toggle High Quality (Fant vs Linear)"/>
+                <Button
```

Comment:
@jules This doesn't save anything anywhere... If you want only change quality of photo - ok you can add a button for it, but when user clicks it, you should save it only to registry, not anywhere else